src/sql/parts/notebook/notebook.css

```diff
@@ -27,12 +27,12 @@
 .notebookEditor .notebook-info-button {
 	display: inline-block;
 	width: 100%;
-	padding: 4px;
+	padding: 0px;
 	text-align: center;
 	cursor: pointer;
 	padding-left: 15px;
 	background-size: 11px;
 	margin-right: 0.3em;
-	font-size: 13px;
+	font-size: 11px;
```

Comment:
This css class has been modified to support both light and dark theme version. Will be able to see the changes once the PR is approved.